imports/api/activities/activity.js

```diff
@@ -0,0 +1,66 @@
+import { Mongo } from 'meteor/mongo';
+import SimpleSchema from 'simpl-schema';
+import { Schemas } from '../@/schemas';
+
+const Activities = new Mongo.Collection('activities');
+
+Schemas.Activity = new SimpleSchema({
+  _id: Schemas.Id,
+  host: Schemas.Hostname,
+
+  authorId: Schemas.Id,
+  authorName: {type: String},
+
+  title: {type: String},
+  subTitle: {type: String, optional: true},
```

Comment:
I couldn't find the right way to do this, but I'm sure it's possible:

For example field `subTitle` is not optional (required), if it's`isPublicActivity` is true. Could we make it like that? There are a few fields that could be fixed in this case...

Reply:
Found the way to do it! 

https://github.com/longshotlabs/simpl-schema#function-properties

Reply:
It would be like this:

```
subTitle: {type: String, optional: !this.field('isPublicActivity').value}

Reply:
Sure I wasnt totally sure what are required and optional ....

Reply:
subTitle: {type: String, optional: true, custom: () => {if(this.field('isPublicActivity').value) return SimpleSchema.ErrorTypes.REQUIRED}},

doc says define optinal true and set custom so I guess this should work